--- a/chrome/browser/ui/browser_ui_prefs.cc
+++ b/chrome/browser/ui/browser_ui_prefs.cc
@@ -222,7 +222,7 @@
       prefs::kHttpsOnlyModeEnabled, false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
   registry->RegisterBooleanPref(
-      prefs::kHttpsFirstBalancedMode, false,
+      prefs::kHttpsFirstBalancedMode, true,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
   registry->RegisterBooleanPref(prefs::kHttpsFirstModeBundleToastQueued, false);
   registry->RegisterBooleanPref(
